src/main/apisix/README.md

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
The latest ks-console can parse comments now. Please add the tags (<!-- -->) back to hide comments.

![image](https://user-images.githubusercontent.com/4861515/91386453-99ecdf00-e865-11ea-90c2-bf1ef76698a8.png)

Reply:
done